README.md

```diff
@@ -16,6 +16,8 @@ Since there are integration limits within each Slack workspace, it is recommende
 
 ## Setting up for local development
 
+*Note:* macOS users may have to install some root certificates if they encounter an SSL error. See [here](https://stackoverflow.com/a/42098127) for details.
```

Comment:
This can be resolved by simply adding `certifi` to the requirements list, so when the requirements are installed the certificates are automatically added.
```suggestion
```

Reply:
Delegated to @lavenduale